### DISCUSSION TITLE
cyw43_driver pico

### DISCUSSION THREAD

**Discussion by Frida854** [Moddable-OpenSource/moddable#1049] [Q&A] [answered]
Has not succeeded. Tried several times to reinstall, to see if I had overlooked something.

```
poulborg@Poul-MBP helloworld % mcconfig -d -m -p pico
make: *** No rule to make target `/Users/poulborg/pico/pico-sdk/src/rp2_common/cyw43_driver/cyw43_bus_pio_spi.pio', needed by `/Users/poulborg/Projects/moddable/build/tmp/pico /debug/helloworld/cyw43_bus_pio_spi.pio.h'. Stop.
```

I followed macOS Setup
Apple M1
Mac OS 13.2.1

**Comment by mkellner** [comment, marked as answer]
Pico SDK 1.5 made some changes which broke Moddable (Bug #1031)

To continue progress, please use Pico SDK 1.4.

**Reply by Frida854** [reply]
OK now on SDK 1.4, next problem:
When I compile pico_w, this error comes and I can see that arm-none-eabi-objcopy
   is in '/opt/homebrew/bin/arm-none-eabi-objcopy', and 
not in '/usr/local/bin/arm-none-eabi-objcopy'.

What can I do?
```
# library:  zepif.c
/bin/sh: /usr/local/bin/arm-none-eabi-objcopy: No such file or directory
# library:  timeouts.c
make: *** [/Users/poulborg/Projects/moddable/build/tmp/pico/pico_w/debug/lib/cyw43_resource.o] Error 127
make: *** Waiting for unfinished jobs....
poulborg@Poul-MBP helloworld % 
poulborg@Poul-MBP helloworld % 
poulborg@Poul-MBP helloworld % which arm-none-eabi-objcopy
/opt/homebrew/bin/arm-none-eabi-objcopy
poulborg@Poul-MBP helloworld % 
```

**Comment by Frida854** [comment]
Sorry, wrong setting in .zshrc. It's running now.

**Comment by mkellner** [comment]
The Moddable SDK has been updated to support Pico SDK 1.5.0.
